bundles/org.openhab.binding.enocean/src/main/java/org/openhab/binding/enocean/internal/Helper.java

```diff
@@ -0,0 +1,41 @@
+/**
+ * Copyright (c) 2010-2019 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.enocean.internal;
+
+import java.util.Arrays;
+
+/**
+ *
+ * @author Daniel Weber - Initial contribution
+ */
+public class Helper {
+
+    public static byte[] concatAll(byte[] a, byte[]... rest) {
+        int totalLength = a.length;
```

Comment:
Rest can be null, even tough you control all use-cases, I would add a:
```
if (rest == null)
   return a;
```
Even before accessing a.length (that really should not ever be null)